plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -270,6 +274,13 @@ const entries = {
 		'wc-blocks-classic-template-revert-button':
 			'./assets/js/templates/revert-button/index.tsx',
 	},
+	cartAndCheckoutFrontend: {
+		...getBlockEntries( 'frontend.{t,j}s{,x}', cartAndCheckoutBlocks ),
+		blocksCheckout: './packages/checkout/index.js',
+		blocksComponents: './packages/components/index.ts',
+		'mini-cart-component':
+			'./assets/js/blocks/mini-cart/component-frontend.tsx',
```

Comment:
Related to my comment above, I'm not sure to understand why `mini-cart-component` is added as part of the `cartAndCheckoutFrontend` entry while the Mini Cart and Mini Cart Contents block are not.

Reply:
@Aljullu Yes its a good point. I am pretty sure I had a reason at the time because I remember exploring this, but I will have to look again because I can't remember the reason.

Reply:
I've added mini cart to the bundle and don't see any issues :)